EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -332,6 +373,9 @@ public static PsmData CreateOnePsmDataEntry(PeptideSpectralMatch psm, Dictionary
             if (trainingVariables.Contains("PsmCount"))
             {
                 psmCount = sequenceToPsmCount[String.Join("|", psm.BestMatchingPeptides.Select(p => p.Peptide.FullSequence).ToList())];
+                List<int> psmCountList = new List<int> { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 30, 40, 50, 75, 100, 200, 300, 400, 500 };
```

Comment:
I believe you have a reason why divide it using the psmCountList. Can you explain?